Rubberduck.Parsing/Grammar/VBALexer.g4

```diff
@@ -298,8 +300,10 @@ SINGLEQUOTE : '\'';
 UNDERSCORE : '_';
 WS : [ \t];
 GUIDLITERAL : '{' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '}';
-IDENTIFIER :  ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\-] ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; -]*;
+IDENTIFIER :  ~[[\](){}\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\-] ~[[\](){}\r\n\t.,'"|!@#$%^&*\-+:=; -]*;
```

Comment:
I don't think we should be modifying the `IDENTIFIER` here. This is not a legal identifier in the VBA code. Could we use a separate lexer rule for VB6 forms to allow the use of curly braces?

Reply:
The lists specify characters that are *not* legal in VBA identifiers - `~[...]`, which includes braces.